src/TodoList.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import TodoListItem from "./TodoListItem";
+
+function TodoList({ todoList, onRemoveTodo }){
+  return (
+    <ul>
+        {todoList.map(todo => <TodoListItem key={todo.id} todo={todo} onRemoveTodo={onRemoveTodo} />)}
+    </ul>
```

Comment:
fyi, you can use a parentheses to turn a multi-line return value into an implicit return. It makes the function a bit more readable.

```javascript
#example

{todoList.map((todo) => (
  <ToDoListItem key={todo.id} todo={todo} onRemoveTodo={onRemoveTodo} />
))}
```

Reply:
Oh I see - thanks! That was helpful!